docs/src/trustedClient.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import LiveEdit from './LiveEdit';
+
+const trustedclient = `
+<TrustedClients />
```

Comment:
once we name it trusted client, we can use it only for trusted clients. we cant use it for partners (for example)
should the components be named without application specificity. basically this is a column of images

Reply:
I believe it all depends on where to keep which component. Our thought was to include trusted clients in serverless-design-system. As you said maybe we can create one more molecule called `ListImage` which can wrap a set of images.